ccloud/docs/index.rst

```diff
@@ -89,8 +89,8 @@ Setup
 
       # BROKER ENDPOINT
       ccloud kafka cluster list
-      ccloud kafka cluster use
-      ccloud kafka cluster describe
+      ccloud kafka cluster use <Id>
+      ccloud kafka cluster describe <Id>
```

Comment:
```suggestion
      ccloud kafka cluster describe <cluster id>
```

Reply:
I intentionally matched the ccloud help text.  WDYT?

Reply:
Sure, let's match ccloud help text.  In that case, can we make `Id` lowercase?

Here is the CLI:

```
ccloud kafka cluster describe
Error: accepts 1 arg(s), received 0
Usage:
  ccloud kafka cluster describe <id> [flags]
```

Reply:
Thanks, fixed... i'm losing my mind because I remember thinking it was strange they were CamelCase when I looked at the help text.   🤷